src/right-field/RightField.jsx

```diff
@@ -240,24 +253,35 @@ async function executeQuery(query, eventEmitter) {
  * A function that given a QueryType processes every result as a stream.
  *
  * @param {QueryType} execution a query execution
+ * @param {query} query the query which is being executed
  * @param {EventEmitter} eventEmitter an EventEmitter that listens to and emits UI state changes.
+ * @param {Function} adder function that sets UI state of the query result
  */
-async function handleQueryExecution(execution, eventEmitter) {
+async function handleQueryExecution(execution, query, eventEmitter, resultAdder) {
```

Comment:
`resultAdder` is not in the docs.

Reply:
This name was outdated in the documentation, I have updated it in [commit e57020e](https://github.com/SolidLabResearch/generic-data-viewer/pull/42/commits/e57020ec88726cd6438ed3c708f54a09b23710d4)